src/components/AnalysisPage/DisplayStatistic.jsx

```diff
@@ -8,7 +8,8 @@ const DisplayStatistic = ({ title, value }) => {
       return "--";
     }
 
-    if (title === "# of Shells" || title === "# of Boundary Edges") {
+    const valueString = value.toString();
+    if (!valueString.includes(".")) {
```

Comment:
Good change! I didn't llike how we were checking for specific titles before. This is much easier to maintain if we add new fields